modules/core/src/main/java/org/apache/ignite/internal/managers/discovery/ClusterMetricsImpl.java

```diff
@@ -379,12 +379,12 @@ public ClusterMetricsImpl(GridKernalContext ctx, long nodeStartTime) {
 
     /** {@inheritDoc} */
     @Override public float getBusyTimePercentage() {
-        return 1 - getIdleTimePercentage();
+        return Math.max(100 - getIdleTimePercentage(), 0);
     }
 
     /** {@inheritDoc} */
     @Override public float getIdleTimePercentage() {
-        return getTotalIdleTime() / (float)getUpTime();
+        return Math.min((float)100 * getTotalIdleTime() / getUpTime(), (float)100.);
```

Comment:
Let's remove a multiplier here to leave public contract unchanged.

Reply:
Again, the method is called " Percentage"